src/components/StoreSearch/StoreSearch.js

```diff
@@ -1,19 +1,24 @@
 import React from 'react';
 import { connect } from 'react-redux';
-import { searchStores } from '../../actions/yelpAPI';
+import {
+  searchStores,
+  setUserLocation,
+  searchStoresWithLocation,
+} from '../../actions/yelpAPI';
 
 import StoreResult from '../StoreResult';
 
+import './StoreSearch.css';
+
 export class StoreSearch extends React.Component {
-  getPosition(options) {
-    return new Promise(function(resolve, reject) {
-      navigator.geolocation.getCurrentPosition(resolve, reject, options);
-    });
+  componentDidMount() {
+    const { dispatch } = this.props;
+    dispatch(setUserLocation());
   }
 
   renderResults() {
     if (this.props.error) {
-      return <strong>{this.props.error}</strong>;
+      return <strong>{this.props.error.message}</strong>;
     }
 
     const stores = this.props.stores.map(store => (
```

Comment:
If the backend API returns `null` then this line is going to fail. You should guard it with an `if (this.props.stores) ...` Otherwise you end up with ugly react typeerror messages.